lib/generator.js

```diff
@@ -20,12 +20,25 @@ module.exports = function(locals) {
   function generate(path, posts, options = {}) {
     options.archive = true;
 
-    result.push(...pagination(path, posts, {
+    const pages = pagination(path, posts, {
       perPage,
       layout: ['archive', 'index'],
       format: paginationDir + '/%d/',
+      explicitPaging: (config.archive_generator.explicit_paging || config.archive_generator.overwrite_latest || false),
```

Comment:
Personally, I think `overwrite_latest` should not be added here, as this may confuse users. It makes it ambiguous whether to enable `explicit_paging`